### PR TITLE
fix8-package

### PR DESCRIPTION
**Pull request by najam-tariq**
This PR attempts to address #147

- modified file paths to fit the package structure
- new way of execution:
  - make a virtual env
  - run command "pip install ."
  - enter "fix8-gui" in the terminal

TODO:
- Implement a fallback mechanism? (in case we want to support the older way of execution)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+[tool.setuptools]
+packages = ["fix8"]
+package-dir = {"fix8" = "src"}
+
+[tool.setuptools.package-data]
+"fix8" = ["*.py", ".images/*", "*.xml"]
+
+[project.scripts]
+fix8-gui = "fix8.fix8:main"
```
Comment (nalmadi) [inline pyproject.toml:60]: Cab we make it just Fix8, please?

**Review by nalmadi — CHANGES_REQUESTED** [review]
Thank you for the excellent work, can we update the readme file in this PR as well?

File: pyproject.toml (hunk 1 above)
Comment (najam-tariq) [reply pyproject.toml]: Done!

**Comment by najam-tariq** [comment]
> Thank you for the excellent work, can we update the readme file in this PR as well?

Done as well!

**Review by nalmadi — APPROVED** [review]
Just got a chance to test this on my side, and everything worked with Python 3.11

Thanks @najam-tariq!

**Pull request merged**